dashboards-observability/.cypress/integration/2_notebooks.spec.js

```diff
@@ -75,12 +75,7 @@ describe('Testing notebooks table', () => {
   it('Displays error toast for invalid notebook name', () => {
     cy.get('.euiButton__text').contains('Create notebook').click();
     cy.wait(delay);
-    cy.get('.euiButton__text')
-      .contains(/^Create$/)
-      .click();
-    cy.wait(delay);
-
-    cy.get('.euiToastHeader__title').contains('Invalid notebook name').should('exist');
```

Comment:
I think we should keep the invalid notebook toast here. Any reason we removed this?

Reply:
Removed this because now the create button will not be clickable unless they have entered in some non-empty string. This is to keep the page on parity with the application analytics create page, as I was thinking to keep a consistent experience across the plugin, but I can change it back if folks think that's not appropriate. @ps48 @mengweieric